### PR TITLE
7.0 package repairs

### PR DESCRIPTION
**Pull request by dzearing**
This brings all packages up to 7.0 and fixes some issues we were seeing on mac builds.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9424)

### CODE REVIEW

**Comment by ecraig12345** [comment]
You'll probably need to add change files to make this actually publish.

File: packages/dashboard/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@uifabric/dashboard",
-  "version": "0.58.2",
+  "version": "0.59.0",
```
Comment (ecraig12345) [inline packages/dashboard/package.json]: Why is this package back??? The entire dashboard package is supposed to be gone!

File: packages/react-cards/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@uifabric/react-cards",
-  "version": "0.5.0",
+  "version": "0.6.0",
```
Comment (ecraig12345) [inline packages/react-cards/package.json]: Why the manual version bump?

**Comment by ecraig12345** [comment]
Oh and please re-enable `ensureConsistentVersions` in rush.json...Anna turned it off a couple months ago for some reason and I missed it...

File: packages/foundation/src/createComponent.tsx [hunk 3]
```diff
@@ -77,7 +77,7 @@ export function createComponent<
     return view(viewProps);
   };
 
-  result.displayName = options.displayName || view.name;
+  result.displayName = options.displayName;
```
Comment (JasonGore) [inline packages/foundation/src/createComponent.tsx]: Did this change creep in from your other PR?

**Comment by ecraig12345** [comment]
David had to go to a meeting, so I'm finishing this up in a different PR.

**Pull request closed without merging**